spec/paperclip/storage/s3_spec.rb

```diff
@@ -971,10 +1113,12 @@ class DummyCredentialProvider; end
         before do
           object = stub
           @dummy.avatar.stubs(:s3_object).returns(object)
-          object.expects(:write).with(anything,
-                                      content_type: "image/png",
-                                      acl: :public_read,
-                                      metadata: { "color" => "red" })
+
+          object.expects((defined?(::Aws) ? :upload_file : :write))
+            .with(anything,
+                  content_type: 'image/png',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.